diff --git a/core/main.mk b/core/main.mk
index bee591d..869599e 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -273,7 +273,7 @@ TARGET_BUILD_JAVA_SUPPORT_LEVEL := platform
 
 # -----------------------------------------------------------------
 # The pdk (Platform Development Kit) build
-include build/core/pdk_config.mk
+#include build/core/pdk_config.mk
 
 # -----------------------------------------------------------------
 ###
@@ -482,10 +482,42 @@ ifneq ($(dont_bother),true)
 # Include all of the makefiles in the system
 #
 
+subdir_makefile_dirs := abi bionic bootable build device external hardware hybris libcore system
+
 # Can't use first-makefiles-under here because
 # --mindepth=2 makes the prunes not work.
+
 subdir_makefiles := \
-	$(shell build/tools/findleaves.py --prune=$(OUT_DIR) --prune=.repo --prune=.git $(subdirs) Android.mk)
+./frameworks/base/Android.mk \
+./frameworks/native/cmds/dumpstate/Android.mk \
+./frameworks/native/cmds/dumpsys/Android.mk \
+./frameworks/native/libs/binder/Android.mk \
+./frameworks/native/libs/gui/Android.mk \
+./frameworks/native/libs/ui/Android.mk \
+./frameworks/native/opengl/libagl/Android.mk \
+./frameworks/native/opengl/libs/Android.mk \
+./frameworks/native/services/powermanager/Android.mk \
+./frameworks/native/services/sensorservice/Android.mk \
+./frameworks/native/services/surfaceflinger/Android.mk \
+./frameworks/native/services/batteryservice/Android.mk \
+./frameworks/native/cmds/servicemanager/Android.mk \
+./frameworks/opt/emoji/Android.mk \
+./frameworks/av/camera/Android.mk \
+./frameworks/av/media/libmedia/Android.mk \
+./frameworks/av/media/libstagefright/Android.mk \
+./frameworks/av/media/CedarX-Projects/Android.mk \
+./frameworks/native/services/connectivitymanager/Android.mk \
+./frameworks/av/drm/libdrmframework/Android.mk \
+./frameworks/av/drm/common/Android.mk \
+./frameworks/av/services/camera/libcameraservice/Android.mk \
+./frameworks/av/media/libmediaplayerservice/Android.mk \
+./prebuilts/gcc/darwin-x86/arm/arm-linux-androideabi-4.6/Android.mk \
+./prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.6/Android.mk \
+./prebuilts/ndk/Android.mk \
+./prebuilts/tools/Android.mk \
+$(shell build/tools/findleaves.py --prune=out --prune=.repo --prune=.git $(subdir_makefile_dirs) Android.mk)
+
+# End of hybris mods
 
 $(foreach mk, $(subdir_makefiles), $(info including $(mk) ...)$(eval include $(mk)))
 
diff --git a/core/product_config.mk b/core/product_config.mk
index e4b68cf..9eee2a4 100644
--- a/core/product_config.mk
+++ b/core/product_config.mk
@@ -166,7 +166,7 @@ endif # unbundled_goals
 ifeq ($(HOST_OS),linux)
 # ... or if the if the option is already set
 ifeq ($(WITH_HOST_DALVIK),)
-  WITH_HOST_DALVIK := true
+  WITH_HOST_DALVIK := false
 endif
 endif
 
diff --git a/core/tasks/apicheck.mk b/core/tasks/apicheck.mk
index 00b78b9..63f8114 100644
--- a/core/tasks/apicheck.mk
+++ b/core/tasks/apicheck.mk
@@ -18,7 +18,7 @@
 #
 
 # skip api check for PDK buid
-ifeq (,$(filter true, $(WITHOUT_CHECK_API) $(TARGET_BUILD_PDK)))
+ifeq (,$(filter true, true $(WITHOUT_CHECK_API) $(TARGET_BUILD_PDK)))
 
 .PHONY: checkapi
 
diff --git a/envsetup.sh b/envsetup.sh
index eac3107..bde3243 100755
--- a/envsetup.sh
+++ b/envsetup.sh
@@ -1,3 +1,5 @@
+
+
 function hmm() {
 cat <<EOF
 Invoke ". build/envsetup.sh" from your shell to add the following functions to your environment:
